db/data_migration/20161222102639_reslug_news_faq.rb

```diff
@@ -0,0 +1,4 @@
+article = NewsArticle.find(688810)
+document = article.document
+
+document.update_attributes(slug: "Information-about-the-UK-leaving-the-EU")
```

Comment:
Zendesk ticket has been updated saying it should be lowercase: https://govuk.zendesk.com/agent/tickets/1480095